### PR TITLE
Add message when no friends are found in the invites page

### PR DESCRIPTION
**Pull request by gmethvin**
@2is10 Also added handlebars so we can start migrating to that.

### CODE REVIEW

File: kifi-backend/modules/shoebox/public/index.html [hunk 1]
```diff
@@ -321,6 +321,7 @@ <h3 class="collection" data-template data-id="{{id}}" style="display:none">
 							</li>
 						</ul>
 						<img class="invite-friends-loading" src="assets/img/wait.gif">
+						<div class="no-results">
```
Comment (2is10) [inline kifi-backend/modules/shoebox/public/index.html]: needs an end tag

File: kifi-backend/modules/shoebox/public/js/main.js [hunk 2]
```diff
@@ -694,6 +695,14 @@ $(function() {
 				friendsShowing.length = 0;
 			}
 			friendsShowing.push.apply(friendsShowing, friends);
+			var $noResults = $nwFriends.find('.no-results').hide();
+			if (!friendsShowing.length) {
+				$noResults.html(noResultsTmpl({ filter: search, network: network })).show();
+				$noResults.find('.refresh-friends').click(function () {
+					$('<form method="post">').attr('action', wwwDomain + '/friends/invite/refresh').submit();
```
Comment (2is10) [inline kifi-backend/modules/shoebox/public/js/main.js]: This doesn’t need to be attached to the document?

File: kifi-backend/modules/shoebox/public/js/main.js [hunk 3]
```diff
@@ -694,6 +695,14 @@ $(function() {
 				friendsShowing.length = 0;
 			}
 			friendsShowing.push.apply(friendsShowing, friends);
+			var $noResults = $nwFriends.find('.no-results').hide();
+			if (!friendsShowing.length) {
+				$noResults.html(noResultsTmpl({ filter: search, network: network })).show();
```
Comment (2is10) [inline kifi-backend/modules/shoebox/public/js/main.js]: Should call `.empty()` before `.html(...)` to ensure that any event handlers previously bound to descendants of `$noResults` are not leaked.

**Comment by 2is10** [comment]
Yay! Let the migration begin. Thanks.

File: kifi-backend/modules/shoebox/public/js/main.js (hunk 2 above)
Comment (gmethvin) [inline kifi-backend/modules/shoebox/public/js/main.js]: It works without but I'll add it to the document to be sure.

Comment (2is10) [inline kifi-backend/modules/shoebox/public/js/main.js]: Thanks. According to [this](http://stackoverflow.com/questions/6009500/jquery-ajaxform-form-is-not-submitted-if-it-is-removed-from-dom-in-beforesubmit) it works in Chrome but not FF or IE.

**Pull request merged**